analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ShouldImplementExportedInterfaces.System.Composition.cs

```diff
@@ -1,7 +1,13 @@
-﻿using System.Collections.Generic;
-using System.Composition;
+﻿using System.Composition;
 
-[Export(typeof(List<>))] // Compliant, FN - Foo<T> should derive from List<T>
```

Comment:
My understanding of the [issue ](https://github.com/SonarSource/sonar-dotnet/issues/5641) that this PR is fixing is that the case with the generics is an FP.
Wouldn't be better to leave this example for the sake of documentation?